### DISCUSSION TITLE
如何实现在第一列加上排序的icon啊，自定义icon不太行，获取不到当前行的index 啊

### DISCUSSION THREAD

**Discussion by qiaogaolong** [antvis/S2#2088] [General]
就比如 第一行 是一个自定义的图标
第二行是一个自定义的图标
第三行是一个自定义的图标
其余的都是另外的图标展示

**Comment by lijinke666** [comment]
![image](https://github.com/antvis/S2/assets/21015895/8a5d4786-e974-471d-9e76-34354ff4e237)


https://s2.antv.antgroup.com/manual/advanced/custom/custom-icon#%E8%87%AA%E5%AE%9A%E4%B9%89%E8%A1%8C%E5%88%97%E5%A4%B4-icon-%E7%A4%BA%E4%BE%8B
